Documentation/PackedAssetBuilds.md

```diff
@@ -18,47 +46,94 @@ Inside the installer select the version to use.  Install version **2021.1.11.793
 
 > REBOOT (or logout / login). Otherwise, the environment settings won't get picked up for any builds in Visual Studio. They will only apply to command-line builds, and only for any command-line windows that have been opened after the installer finishes.
 
-### Build profile build and process assets
+### Build non-monolithic profile builds of the Engine
 
-Build and run MPS as per the [README.md](../README.md) and ensure all assets are built.
+Build a regular profile build of the game as per the [README.md](../README.md) in an engine-centric way. Be sure to build from the source engine and not the installer. Here is an example:
 
-### Test the profile build
+Clone the engine from source.
 
-* Open the game in editor
-    * load `NewStarBase` level
-    * Verify that game can launch and connect to local server from editor
-* Validate local game launcher can connect to local server
+```shell
+C:\git> git clone https://github.com/o3de/o3de
+```
+
+Navigate to `C:\git\o3de`.
+Create a build folder for non-monolithic build of the engine, for example `C:\git\o3de\build_non_mono`.
+
+```shell
+C:\git\o3de> mkdir build_non_mono
+```
+> The location and the name of the build folder is optional.
+
+Configure the engine in a non-monolithic mode with o3de-multiplayersample project.
+
+```shell
+C:\git\o3de\build_non_mono> cmake .. -DLY_MONOLITHIC_GAME=OFF -DLY_PROJECTS="C:\git\o3de-multiplayersample"
```

Comment:
Are you using engine centric to avoid having to run `get_python.bat`, register the engine and the project?

I kinda wish you could keep the project outside the engine folder to avoid git thinking there are a ton of changes
```shell
# get the engine and python
git clone https://github.com/o3de/o3de  c:\git\o3de
c:\git\o3de\python\get_python.bat

# get the project and tell it to use the engine
git clone https://github.com/o3de/o3de-multiplayersample c:\git\o3de-multiplayersample
c:\git\o3de\scripts\o3de.bat edit-project-properties -pp c:\git\o3de-multiplayersample --engine-path c:\git\o3de --user

# configure and build profile non-mono
cd c:\git\o3de-multiplayersample
cmake --preset windows-default
cmake --build build/windows --target Editor AssetBundler --config profile

# configure and build mono profile
cmake --preset  windows-mono-default  -DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0 -DLY_PROJECTS="C:\git\o3de-multiplayersample"
cmake --build build/windows_mono --target MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher MultiplayerSample.UnifiedLauncher --config profile

# Process assets
c:\git\o3de-multiplayersample\build\windows\bin\profile\AssetProfilerBatch.exe

# Bundle assets - there's probably a way to pass in the seed lists from the command line
c:\git\o3de-multiplayersample\build\windows\bin\profile\AssetBundlerBatch.exe --foo=path/to/seedlist --bundle=path/to/engine.pak
c:\git\o3de-multiplayersample\build\windows\bin\profile\AssetBundlerBatch.exe --foo=path/to/seedlist --bundle=path/to/game.pak

# Copy binaries and assets into release folder
cp blah blah blah
```

Reply:
The project is outside of the engine: C:\git\o3de\build_non_mono versus C:\git\o3de-multiplayersample. But I am keeping engine-centric otherwise, you have to add steps on registering a new install layout to the steps, which would add a lot of complexity to these steps.